### DISCUSSION TITLE
Oxc linter has release schedule?

### DISCUSSION THREAD

**Discussion by pumano** [oxc-project/oxc#6523] [General]
Folks I'm very impressed about `oxlint`, because I have large monorepo with 90+ libs (~10k files) and eslint is pain point for me and our 40 engineers team (around ~12-14 mins in CI pipeline without cache) and when I get lint feedback in less than second it's very impressive!

I'm testing `oxlint` against that monorepo and found some bugs, and I want to got updates more often (maybe nightly pre-releases?) or maybe you tell me how I can build oxlint locally (and vscode extention) for testing and fill the issues only when I test it against latest code.

**Comment by nrayburn-tech** [comment]
Nightly releases of oxlint on NPM would be convenient for developing IDE extensions without having to build the language server.  I'm sure other people could find use cases too.

In any case, below is the general idea of how to build the VS Code extension (and pre-requisites).
1. Follow the steps here https://oxc.rs/docs/contribute/development.html to get all the installed tools.
2. Navigate to https://github.com/oxc-project/oxc/tree/main/editors/vscode in your local terminal.
3. Run `pnpm install` to install dependencies.
4. Run `pnpm build` to compile the language server, compile the extension, and package it all together in a vsix file.
5. Then you can install the vsix file however you want.

Some ways to install a vsix.
1. Run `pnpm install-extension` which executes `code --install-extension oxc_language_server.vsix --force`.  This only works if you have `code` available on your path.  If you use vscode insiders, you'll want to use `code-insiders` instead.  If you don't have either on your path then this won't work.
2. Open VS Code, go to the extensions tab, select the three dots, and pick install from vsix and select the vsix that you created.

**Reply by pumano** [reply]
Thank you @nrayburn-tech! My first problem here: I previously not install rust via recommended way and try to install via brew and got problems: `rustfmt` just not working in that type installation out of the box and I got errors when try to `just ready`.

Currently I compile properly and install fresh vscode extention

**Comment by Boshen** [comment]
We release roughly once a week.
